eventbus/jetstream/sensor/trigger_conn.go

```diff
@@ -28,6 +28,13 @@ type JetstreamTriggerConn struct {
 	evaluableExpression  *govaluate.EvaluableExpression
 	deps                 []eventbuscommon.Dependency
 	sourceDepMap         map[string][]string // maps EventSource and EventName to dependency name
+	recentMsgsByID       map[string]*Msg     // prevent re-processing the same message as before (map of msg ID to time)
+	recentMsgsByTime     []*Msg
+}
+
+type Msg struct {
```

Comment:
`msg`?

Reply:
Oh, I see: lower case it to keep it only accessible within the package? sure, good idea